src/BUILD.plz

```diff
@@ -48,3 +48,12 @@ filegroup(
     binary = True,
     visibility = ["//:install"],
 )
+
+# On Linux, we should produce statically linked binaries. Emit a test to verify this is the case.
+if is_platform(os = "linux"):
+    gentest(
+        name = "static_test",
+        test_cmd = "ldd $DATA && exit 1 || exit 0",
```

Comment:
Clever! Perhaps we should be more explicit though (`file $DATA | grep 'statically linked'`)?

Reply:
I tried that first out :) It failed on most of our Docker images that don't have `file` in them, this seems like a slightly more common denominator.